tests/unit_tests/dlopen_source/test_dlopen_source.py

```diff
@@ -0,0 +1,57 @@
+import openmc
+import pytest
+import os
+
+# compile the external source
+def compile_source():
+    # needs a more robust way to know where the
+    # Makefile is
+    status  = os.system('cp /usr/local/share/Makefile .')
```

Comment:
Use one of the functions from `shutil` (copyfile or copy2) rather than os.system

Reply:
Ok